src/OrchardCore.Modules/OrchardCore.Placements/Settings/PlacementContentTypePartDefinitionDriver.cs

```diff
@@ -17,46 +18,91 @@ public PlacementContentTypePartDefinitionDriver(IStringLocalizer<PlacementConten
+            return Initialize<PlacementSettingViewModel>("PlacementSettings", model =>
             {
                 var contentType = contentTypePartDefinition.ContentTypeDefinition.Name;
                 var partName = contentTypePartDefinition.Name;
+                var partDisplayName = contentTypePartDefinition.DisplayName();
                 var displayName = contentTypePartDefinition.ContentTypeDefinition.DisplayName;
 
-                model.ContentSettingsEntries.Add(
-                    new ContentSettingsEntry
+                var partTypeName = contentTypePartDefinition.PartDefinition.Name;
+
+                var shapeType = partTypeName;
+                var differentiator = partName;
+
+                model.PlacementSettingEntries.Add(
+                    new PlacementSettingEntry
                     {
-                        ShapeType = partName,
+                        ShapeType = shapeType,
+                        Differentiator = differentiator,
                         ContentType = contentType,
-                        Description = S["Placement for the {0} part in a {1} type", partName, displayName]
+                        Description = S["{0} part in a {1} type", partDisplayName, displayName]
                     });
 
-                model.ContentSettingsEntries.Add(
-                    new ContentSettingsEntry
+                model.PlacementSettingEntries.Add(
+                    new PlacementSettingEntry
                     {
-                        ShapeType = partName,
+                        ShapeType = shapeType,
+                        Differentiator = differentiator,
                         ContentType = contentType,
                         DisplayType = "Detail",
-                        Description = S["Placement for the {0} part in a {1} type in detail views", partName, displayName]
+                        Description = S["{0} part in a {1} type in detail views", partDisplayName, displayName]
                     });
 
-                model.ContentSettingsEntries.Add(
-                    new ContentSettingsEntry
+                model.PlacementSettingEntries.Add(
+                    new PlacementSettingEntry
                     {
-                        ShapeType = partName,
+                        ShapeType = shapeType,
+                        Differentiator = differentiator,
                         ContentType = contentType,
                         DisplayType = "Summary",
-                        Description = S["Placement for the {0} part in a {1} type in summary views", partName, displayName]
+                        Description = S["{0} part in a {1} type in summary views", partDisplayName, displayName]
+                    });
+
+                model.PlacementSettingEntries.Add(
+                    new PlacementSettingEntry
+                    {
+                        Description = S["-"]
```

Comment:
It's not a super nice technique, but at least it should not call `S` and instead use `new LocalizedString("-")` such that translations can't break it.